flowauth/frontend/cypress/integration/add_new_server.js

```diff
@@ -14,95 +14,96 @@ describe("Server management", function() {
   it("Add server name with space", function() {
     cy.get("#new").click();
     // adding username with space
-    cy.get("#name").type("Server ");
+    cy.get("#name").type("Server ", { force: true });
```

Comment:
What's the purpose of `force: true` here (and below)?

Reply:
Works around cypress trying to scroll the thing into view, but it ending up hidden because their scrolling algorithm is bad. This types (or whatever) into the box even if it isn't visible - I'd add a comment to this effect, but we end up having to do it pretty much universally across the tests, so is already mentioned elsewhere.

Reply:
Yeah I was going to say the same thing. Would be nice to have a comment but since it's everywhere it's already fairly visible (just not entirely clear why if you don't see that comment). A bit annoying that we have to do this everywhere but seems to be necessary. Out of interest, is this a known bug that they are aware of and that might be fixed some time in the future?

Reply:
Yeah, see https://github.com/cypress-io/cypress/issues/871

They are apparently working on it (or were in December last year).